lms/djangoapps/teams/static/teams/js/views/team_card.js

```diff
@@ -112,6 +114,15 @@
                 return this.model;
             },
 
+            getMemberships: function () {
+                if (this.model.has('team')) {
```

Comment:
I noticed this duplicates logic in `teamModel`. This could be a one-liner: `return this.teamModel().get('membership')`, which will make the eventual card refactor easier too.